src/angular/directives/flex-copy.directive.ts

```diff
@@ -21,7 +21,7 @@ export class FlexCopyDirective {
         for (let i = 0; i < selection.rangeCount; i++) {
             const range = selection.getRangeAt(i);
             const text = range.toString();
-            copyText += text;
+            copyText += text.substring(0, text.search(/\r?\n/));
```

Comment:
How does this work if new line isn't found? Seems like it would substring(0, -1) which would be nothing. Did you test this?

Reply:
@kspearrin I’m sorry, of course. I’ll push an updated version of that commit to maintain the original behavior for lines not containing newlines at all.